### PR TITLE
test: add sealer alpha mount/umount e2e test

### PR DESCRIPTION
**Pull request by Stevent-fei**
### Describe what this PR does / why we need it
add sealer alpha mount/umount e2e test

### CODE REVIEW

File: .github/workflows/e2e-test-mount.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: Sealer-Test-Mount
```
Comment (kakaZhou719) [inline .github/workflows/e2e-test-mount.yml:1]: Mount is a sub command for image ,i think we only need add it to image e2e test

File: test/sealer_mount_test.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+		It("start to mount and umount", func() {
+
+			By("start to mount a image")
+			mountCmd := fmt.Sprintf("%s alpha mount %s -d", settings.DefaultSealerBin, settings.TestImageName)
+			testhelper.RunCmdAndCheckResult(mountCmd, 0)
+
+			By("start to umount a image")
+			umountCmd := fmt.Sprintf("%s alpha umount %s -d", settings.DefaultSealerBin, settings.TestImageName)
+			testhelper.RunCmdAndCheckResult(umountCmd, 0)
```
Comment (kakaZhou719) [inline test/sealer_mount_test.go:43]: could we check more details of mount, such as mounted dir exiting , or image file.

File: test/sealer_images_test.go [hunk 3]
```diff
@@ -112,4 +114,65 @@ var _ = Describe("sealer images module", func() {
...
+			testhelper.CheckBeTrue(build.CheckIsImageExist(settings.TestK8s118))
+		})
+
+		It("start to mount and umount", func() {
+
+			By("start to mount multiple images")
+			mountCmd := fmt.Sprintf("%s alpha mount %s %s %s -d", settings.DefaultSealerBin, settings.TestImageName, settings.TestK8s120, settings.TestK8s118)
+			testhelper.RunCmdAndCheckResult(mountCmd, 0)
+
+			err := filepath.WalkDir("/var/lib/containers/storage/overlay", func(path string, d fs.DirEntry, err error) error {
```
Comment (kakaZhou719) [inline test/sealer_images_test.go:153]: why "/var/lib/containers/storage/overlay" , make it const or get it from imageengine ?

File: test/sealer_images_test.go [hunk 4]
```diff
@@ -112,4 +114,65 @@ var _ = Describe("sealer images module", func() {
...
+					testhelper.CheckErr(err)
+				}
+				return nil
+			})
+			if err != nil {
+				return
+			}
+
+			By("start to umount multiple images")
+			umountCmd := fmt.Sprintf("%s alpha umount %s %s %s -d", settings.DefaultSealerBin, settings.TestImageName, settings.TestK8s120, settings.TestK8s118)
```
Comment (Starnop) [inline test/sealer_images_test.go:164]: add test case for `mount multi app images`

**Review by kakaZhou719 — APPROVED** [review]
LGTM

**Pull request merged**